go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/mr-tron/base58 v1.2.0
 	github.com/nspcc-dev/dbft v0.0.0-20210721160347-1b03241391ac
 	github.com/nspcc-dev/go-ordered-json v0.0.0-20220111165707-25110be27d22
+	github.com/nspcc-dev/neo-go/examples/nft-nd-nns v0.0.0-20220118092524-efdb66b159f0
```

Comment:
But why do we need it?

Reply:
Our RPC client imports some constants and structs to work with NNS contract.

Reply:
Right, forgot about it.